linux_pipeline/Jenkinsfile_developer_patch_validation

```diff
@@ -184,7 +186,7 @@ pipeline {
                             --build_name $BUILD_NAME --build_number "${BUILD_NUMBER}${BRANCH_NAME}" \
                             --smb_share_username $USERNAME --smb_share_password $PASSWORD \
                             --smb_share_url $SMB_SHARE_URL --vm_user_name $OS_TYPE \
-                            --os_type $OS_TYPE
+                            --os_type $OS_TYPE --flavour $AZURE_VM_SIZE --resource_group $AZURE_REGION
```

Comment:
it s --resource_location, not resource_group. I suppose also resource group must be added as a parameter? or it s the same one?